lib/feedback.js

```diff
@@ -197,17 +197,27 @@ function feedback(system) {
 								}
 
 								try {
+									function handleResult(result) {
```

Comment:
can you make this a class function?

Reply:
@Julusian if this gets addressed, I'm otherwise good with this PR

Reply:
done. Ive pushed the logic to inside of setStyle
I need to retest it, and look into an upgradeScript helper, then I shall merge it